sqlobject/src/main/java/org/jdbi/v3/sqlobject/locator/ClasspathSqlLocator.java

```diff
@@ -0,0 +1,23 @@
+package org.jdbi.v3.sqlobject.locator;
+
+import org.jdbi.v3.core.config.ConfigRegistry;
+import org.jdbi.v3.sqlobject.internal.SqlAnnotations;
+
+import java.lang.reflect.Method;
+import java.util.function.Function;
+
+/**
+ * @see org.jdbi.v3.core.locator.ClasspathSqlLocator
```

Comment:
Wasn't sure what more could be said here, I can just copy the doc over if you prefer.